core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -32,11 +32,26 @@ oppia.factory('UrlInterpolationService', [
       }
     };
 
-    var getCachePrefixedUrl = function(resourcePath) {
+    var getPrefixedUrl = function(resourcePath) {
       validateResourcePath(resourcePath);
       return GLOBALS.ASSET_DIR_PREFIX + resourcePath;
     };
 
+    var getUrlWithSlug = function(resourcePath) {
```

Comment:
Ah yes, good point. I think this function is complex enough that it should probably be tested, so yes, I think that what you suggest is fine. Thanks for checking!